fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java

```diff
@@ -506,6 +507,8 @@ public boolean visit(java.lang.String elementName, int elementIndex, Money money
             p.setResourceType(resourceType);
             p.setName(searchParamCode);
             p.setValueNumber(money.getValue().getValue());
+            p.setValueNumberLow(NumberParmBehaviorUtil.generateLowerBound(money.getValue().getValue()));
+            p.setValueNumberHigh(NumberParmBehaviorUtil.generateUpperBound(money.getValue().getValue()));
```

Comment:
should this be the NewNumberParmBehaviorUtil ? does that have the lower/upper bound stuff?

Reply:
Good catch - changed references to NewNumberParmBehaviorUtil (methods are same as old class)